openmetadata-ui/src/main/resources/ui/.eslintrc

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2021 Collate
+ *  Copyright 2022 Collate.
```

Comment:
@chirag-madlani lets change it 2023 no?

Reply:
I kept it dynamic, It will change once the calendar changes